CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ All notable changes to this project will be documented in this file.
 -   \[INTERNALS\] Allow the use of magic methods to describe container methods.
 -   \[DEVELOPER\] Added an improved traceback to the developer-mode errors for errors in function calls.
 -   \[DEVELOPER\] Added an environment variable to globally activate developer-mode for errors.
+-   #2256 Add C and Fortran support for list method `clear()`.
```

Comment:
Please place messages for users above messages for devs. If possible it is also good to group similar information so this should go here:
https://github.com/pyccel/pyccel/blob/b249b269a9595b41b6414bd498d00377e41f379b/CHANGELOG.md?plain=1#L34-L35

Reply:
fixed